odoo/tests/common.py

```diff
@@ -530,6 +530,7 @@ def _chrome_start(self):
             '--user-data-dir': self.user_data_dir,
             '--disable-translate': '',
             '--window-size': '1366x768',
+            '--remote-debugging-address': HOST,
```

Comment:
The goal of the change is that then it works fine even if the value of HOST is changed. We can change the config IP in only one place instead of several.

Reply:
I agree